utils/simplecache/config.go

```diff
@@ -0,0 +1,31 @@
+package simplecache
+
+import (
+	"time"
+
+	validation "github.com/go-ozzo/ozzo-validation/v4"
+
+	configUtils "github.com/ARM-software/golang-utils/utils/config"
+	"github.com/ARM-software/golang-utils/utils/filesystem"
+)
+
+type Config struct {
+	cachefs                 filesystem.FS `mapstructure:"cache_filesystem"`
```

Comment:
this is not really settable from environment and so the filesystem should be stored as a field of the cache